module/Finna/src/Finna/View/Helper/Root/UserListEmbed.php

```diff
@@ -54,6 +54,13 @@ class UserListEmbed extends \Zend\View\Helper\AbstractHelper
      */
     protected $listTable;
 
+    /**
+     * Index start counter
```

Comment:
Laita tarkempi selitys siitä, mihin tuota käytetään, esim. "Counter used to ensure unique id attributes when several lists are displayed"